src/app/pages/home/rolodex/rolodex.page.html

```diff
@@ -1,43 +1,6 @@
-<ion-toolbar>
```

Comment:
We discussed the mechanics of the rolodex, so you should be able to use it properly with the code added back in.

Reply:
I'll work on slowly adding this back in. I need a refresher on the key design points to preserve.